Podcast/DiscoverViewController.swift

```diff
@@ -26,32 +25,13 @@ class DiscoverViewController: ViewController, UITableViewDelegate, UITableViewDa
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .offWhite
-        
-        let tabbedPageViewController = TabbedPageViewController()
-        tabbedPageViewController.searchResultsDelegate = self
-        tabbedPageViewController.searchRequestsDelegate = self
-        let searchResultsController = tabbedPageViewController
-        searchController = UISearchController(searchResultsController: searchResultsController)
-        searchController.searchResultsUpdater = searchResultsController
-        
-        let cancelButtonAttributes: NSDictionary = [NSAttributedStringKey.foregroundColor: UIColor.sea]
-       UIBarButtonItem.appearance().setTitleTextAttributes(cancelButtonAttributes as? [NSAttributedStringKey: Any], for: .normal)
-        
-        searchController.searchBar.showsCancelButton = false
-        searchController.searchBar.searchBarStyle = .minimal
-        searchController.searchBar.placeholder = "Search"
-        navigationItem.titleView = searchController.searchBar
-        searchController.searchBar.sizeToFit()
-        
-        searchController.delegate = self
-        searchController.hidesNavigationBarDuringPresentation = false
-        searchController.dimsBackgroundDuringPresentation = false
-        definesPresentationContext = true
+        title = "Discover"
         
         tableView = UITableView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: view.frame.height), style: .grouped)
```

Comment:
change this to snap kit?

Reply:
For now, this PR is just dealing with search and not discover .. I really only touched discover so I could refactor search out